examples/crowdfunding/package.json

```diff
@@ -4,7 +4,8 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "dependencies": {
-    "algosdk": "^1.8.0"
+    "algosdk": "^1.8.0",
+    "glob": "^7.1.6"
```

Comment:
Why do we need this dependency in crowdfunding?

Reply:
You're right, we don't need this.  I have removed it.